docs/pages/application-access/guides/connecting-apps.mdx

```diff
@@ -124,7 +73,25 @@ An application name should make a valid sub-domain (\<=63 characters, no spaces,
 After Teleport is running, users can access the app at `app-name.proxy_public_addr.com`
 e.g. `grafana.teleport.example.com`. You can also override `public_addr` e.g
 `grafana.acme.com` if you configure the appropriate DNS entry to point to the
-Teleport proxy server.
+Teleport proxy server for self-hosted Teleport instances.
+
+<Admonition type="tip" title="Application URI Domains and TLS for Self-Hosted" scope="oss,enterprise" scopeOnly>
```

Comment:
```suggestion
<Admonition type="tip" title="Application Domain Names and TLS" scope="oss,enterprise" scopeOnly>
```
We won't need "for Self-Hosted" in the title if the admonition displays only for users of self-hosted editions.